sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,711 @@
+
+The `createBehavior()` method can be used to create a `Behavior` that is bound to the element using the `CSSDirective`:
+
+```ts
+class RandomWidth extends CSSDirective {
+  private property = "--random-width";
+  createCSS() {
+    return `width: var(${this.property});`
+  }
+
+  createBehavior() {
+    return {
+      bind(el) {
+        el.style.setProperty(this.property, Math.random() * 100)
+      }
+      unbind(el) {
+        el.style.removeProperty(this.property);
+      }
+    }
+  }
+}
+```
+
+---
+## Design System
+### [Creating a Design System](https://www.fast.design/docs/design-systems/creating-a-component-library#defining-a-design-system)
+
+1. Start with the base components defined in `@microsoft/fast-foundation` and compose them with your own styles:
+
+```ts
+import {
+    Button,
+    buttonTemplate as template,
+} from "@microsoft/fast-foundation";
+import { buttonStyles as styles } from "./special-button.styles";
+
+export const specialButton = Button.compose({
+    baseName: "button",
+    template,
+    styles,
+    shadowOptions: {
+        delegatesFocus: true,
+    },
+});
+
+export const buttonStyles = styles;
+```
+
+
+
+
+2. Export a `provider function` for your community to use in setting everything up:
+
+```ts
+export function provideSpecialDesignSystem(element?: HTMLElement): DesignSystem {
+    return DesignSystem.getOrCreate(element).withPrefix("special");
+}
+```
+
+---
+
+### [Design Tokens](https://www.fast.design/docs/design-systems/design-tokens)
+
+
+[Creating a token](https://www.fast.design/docs/design-systems/design-tokens#create-a-token):
+
+```ts
+import { DesignToken } from "@microsoft/fast-foundation";
+
+// create a design token
+export const specialColor = DesignToken.create<string>("special-color");
+```
+
+
+[Using design tokens in CSS](https://www.fast.design/docs/design-systems/design-tokens#using-design-tokens-in-css):
+```ts
+// use the design token to build styles
+const styles = css`
+  :host {
+    background: ${specialColor};
+  }
+`;
+
+const ancestor = document.querySelector("my-ancestor") as FASTElement;
+const descendent = ancestor.querySelector("my-descendent") as FASTElement;
+```
+
+[Setting values](https://www.fast.design/docs/design-systems/design-tokens#setting-values):
+```ts
+// change the value for a given element
+specialColor.setValueFor(ancestor, "#FFF");
+specialColor.setValueFor(descendent, "#000");
+```
+
+[Getting values](https://www.fast.design/docs/design-systems/design-tokens#getting-values):
+```ts
+// get the value
+specialColor.getValueFor(ancestor); // "#FFF"
+specialColor.getValueFor(descendent); // "#000"
+```
+
+[Deleting values](https://www.fast.design/docs/design-systems/design-tokens#deleting-values):
+```ts
+// unset the value (inherits from ancestor)
+specialColor.deleteValueFor(descendent);
+specialColor.getValueFor(descendent); // "#FFF"
+```
+
+---
+#### Configuring a FAST Design Token:
```

Comment:
An important note here is that this design token (and others exported from FAST) are only available from the `@microsoft/fast-components` package because they're part of an implementation of a design system

Reply:
@nicholasrice Thank you - updated!